api/src/main/java/marquez/db/EventDao.java

```diff
@@ -0,0 +1,37 @@
+package marquez.db;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import java.util.List;
+import marquez.db.mappers.RawLineageEventMapper;
+import org.jdbi.v3.sqlobject.config.RegisterRowMapper;
+import org.jdbi.v3.sqlobject.customizer.Bind;
+import org.jdbi.v3.sqlobject.statement.SqlQuery;
+
+@RegisterRowMapper(RawLineageEventMapper.class)
```

Comment:
Have you looked into using the [`@Json`](https://jdbi.org/#_operation) instead of defining a custom mapper?

> Our PostgresPlugin provides qualified factories that will bind/map the @EncodedJson String to/from json or jsonb-typed columns.

Reply:
Replaced with `LineageEventMapper` that does what I want it to.